cmd/cloud-controller-manager/providers.go

```diff
@@ -22,8 +19,6 @@ limitations under the License.
 
 package main
 
-import (
-	// NOTE: Importing all in-tree cloud-providers is not required when
-	// implementing an out-of-tree cloud-provider.
-	_ "k8s.io/legacy-cloud-providers/gce"
-)
+// NOTE: Importing all in-tree cloud-providers is not required when
+// implementing an out-of-tree cloud-provider. Leaving this empty file
+// here as a reference.
```

Comment:
If  possible we should still reference some import so new developers know that an import to a cloud provider package is required here